client/components/RemoveFromCart.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import { removeFromCart } from "../store/cart";
+
+const RemoveFromCart = (props) => {
+    const dispatch = useDispatch();
+    const productId = props.productId
+    return (
+        <button type="submit" onClick={() => dispatch(removeFromCart(productId))}>Remove from Cart</button>
```

Comment:
Does type "submit" imply that a button is for submitting a form? If it works, doesn't matter, but that was my impression